src/components/Instructions.component.js

```diff
@@ -5,8 +5,8 @@ import './../styles/instructions.css'
 
 class Instructions extends Component {
   render() {
-    let items = this.props.items
-    items = items
+    const items = this.props.items
+    let filtered_items = items
```

Comment:
use `const`

of, in this case, you can just replace this line and the previous one with:
```javascript
const filtered_items = this.props.items
  .filter(...
```